components/core/src/os/net/mod.rs

```diff
@@ -22,3 +22,73 @@ mod imp;
 mod imp;
 
 pub use self::imp::*;
+
+extern crate dns_lookup;
+
+use dns_lookup::{getaddrinfo,
+                 AddrInfoHints};
+use std::io;
+
+// lookup_fqdn returns a vector of fqdn that resolves the provided hostname.
```

Comment:
In Rust, documentation comments on functions/methods use `///` instead of `//`.

Additionally, this is no longer what the function returns 😄

Reply:
Thank you both, updated it.